tests/playwright/sanity/editor-v2.test.ts

```diff
@@ -28,9 +28,10 @@ test.describe( 'Editor v2', () => {
 
 		await wrapper.getByRole( 'button', { name: 'Post Settings' } ).click();
 
-		await editor.page.getByLabel( 'Title', { exact: true } ).fill( 'Test page' );
+		await editor.page.getByLabel( 'Title', { exact: true } ).fill( 'Playwright Test Page' );
 
-		await wrapper.getByRole( 'button', { name: 'Test page' } ).waitFor();
+		await wrapper.getByRole( 'button', { name: 'Playwright Test Page' } ).waitFor();
+		await editor.page.waitForTimeout( 2000 );
```

Comment:
The timeline at the top of the editor was causing unequal screenshots. I have added this timeout, to make sure that the timeline has disappeared.

Reply:
can we wait for selector not to be visible? We should avoid this hardcoded timeouts.

Reply:
I am not sure where the timeline styling is set. I don't know how to get around this.

@dmaziar

Reply:
<img width="176" alt="Screenshot 2023-12-20 at 07 45 11" src="https://github.com/elementor/elementor/assets/94352322/8893e479-bb7b-4371-9c57-ee8ae4a92756">

It is this pink/purple line at the top.

Reply:
@dmaziar Updated with the `isUitStable` method.